diff -rupN ps-printer-app-65b53e2ced957c6d07a6bd7df77631989c3ad263/ps-printer-app.service ps-printer-app-65b53e2ced957c6d07a6bd7df77631989c3ad263_service-user/ps-printer-app.service
--- ps-printer-app-65b53e2ced957c6d07a6bd7df77631989c3ad263/ps-printer-app.service	2021-09-08 12:24:36.000000000 -0500
+++ ps-printer-app-65b53e2ced957c6d07a6bd7df77631989c3ad263_service-user/ps-printer-app.service	2021-09-26 13:02:05.421682952 -0500
@@ -2,6 +2,7 @@
 Description=PostScript Printer Application
 
 [Service]
+User=ps-printer-app
 ExecStart=ps-printer-app server
 ExecStop=ps-printer-app shutdown
 Type=simple
